piyushgarg-dev/piyushgargdev-nextjs
# fix: student lmages Height in student feedback testimonials

## What does this PR do?


Fix the issue of inconsistent height in Student images in Student feedback testimonial section

Fixes #166
![image](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/107617184/9420051f-db93-4ee3-a5b9-16c822f455ae)

![image](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/107617184/dc5a2fb2-8a2b-402e-ab77-91d48d22001d)



## Type of change
- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?
1-Go To Home Page
2-scroll-down to student feedback testimonials section
3-see the Students images
